src/ArrayLayouts.jl

```diff
@@ -52,13 +52,10 @@ else
     const CNoPivot = NoPivot
 end
 
-if VERSION ≥ v"1.11.0-DEV.21"
-    using LinearAlgebra: UpperOrLowerTriangular
+@static if VERSION ≥ v"1.11.0-DEV.21"
```

Comment:
For testing purposes, could you try
```suggestion
@static if VERSION ≥ v"1.10-"
```
and then revert? What's the breakage actually? Is it that some packages need  `AbstractTriangular` beyond `UpperOrLowerTriangular`?

Reply:
It caused a bunch of ambiguities but these have been fixed downstream. I’m not sure it’s worth merging this anymore